src/pages/addProduct/index.tsx

```diff
@@ -0,0 +1,93 @@
+import Container from '@components/container';
+import { Box, Stack, Typography, TextField, Checkbox } from '@mui/material';
+import { useSelector } from 'react-redux';
+
+import * as styles from './addproduct.styles';
+const AddProduct = () => {
+    const dropDown = useSelector((state) => state['dropdowns'].sizes);

```

Comment:
variable name should be sizes & useSelector callback return value should be `state.dropdowns.sizes`

Reply:
resolve